src/Components/CreateImageWizard/utilities/PasswordValidatedInput.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+
+import {
+  FormGroup,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+  InputGroup,
+  InputGroupItem,
+  TextInput,
+  Button,
+  TextInputProps,
+} from '@patternfly/react-core';
+import { EyeIcon, EyeSlashIcon } from '@patternfly/react-icons';
+
+import { checkPasswordValidity } from './useValidation';
+
+import { useAppSelector } from '../../../store/hooks';
+import { selectImageTypes } from '../../../store/wizardSlice';
+
+type ValidatedPasswordInput = TextInputProps & {
+  value: string;
+  placeholder: string;
+  ariaLabel: string;
+  onChange: (event: React.FormEvent<HTMLInputElement>, value: string) => void;
+};
+
+export const PasswordValidatedInput = ({
+  value,
+  placeholder,
+  ariaLabel,
+  onChange,
+}: ValidatedPasswordInput) => {
+  const environments = useAppSelector(selectImageTypes);
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+
+  const { validationState, strength } = checkPasswordValidity(
+    value,
+    environments.includes('azure')
+  );
+  const { ruleLength, ruleCharacters } = validationState;
+
+  const togglePasswordVisibility = () => {
+    setIsPasswordVisible(!isPasswordVisible);
+  };
+
+  const passStrLabel = (
+    <HelperText>
+      <HelperTextItem variant={strength.variant} icon={strength.icon}>
+        {strength.text}
+      </HelperTextItem>
+    </HelperText>
+  );
+
+  return (
+    <FormGroup label="Password" isRequired labelInfo={passStrLabel}>
```

Comment:
I wouldn't say this is a strong password :sweat_smile: could we show the label for Azure targets only? What do you think?

![image](https://github.com/user-attachments/assets/3c8a5228-f9d7-4efa-8d7b-1266bb662b17)

Reply:
As I understand from this conversation -https://redhat-internal.slack.com/archives/C05LNKHBCDA/p1742306355820419?thread_ts=1742303474.431899&cid=C05LNKHBCDA
if we have one condition (all the validation rule are passing-> so the passwrod strengh should show

Reply:
I understand it the same way. Let's keep it in this PR, open an issue and discuss how to handle this in a follow up. Showing "strong password" for weak passwords could be both confusing and misleading for the user, but since we don't have any other criteria for "other targets" passwords, we will need to explore when to show the label.

Reply:
I dont have strong opinion here, but I agree that it is not make sense to see the strong label with just few letters, 
it is very simple fix, to keep it only for Azure targets,
so I am ok to keep this discussion here, and see what @lucasgarfield will say